src/patient/patient-info-cell.component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { concatAgePatientDisplay, usePatientAge } from './patient.resources';
+
+type PatientInfoCellProps = {
+  patient: {
+    name: string;
+    uuid: string;
+  };
+};
+
+const PatientInfoCell: React.FC<PatientInfoCellProps> = ({ patient: { name, uuid } }) => {
+  const { error, isLoading, age } = usePatientAge(uuid);
+  if (isLoading || error) return <>{name}</>;
+  const patient = concatAgePatientDisplay(name, age);
+  return <>{patient}</>;
```

Comment:
Shouldn't this be {name}?

Reply:
it was ok, but i have renamed the variables appropriately to prevent confusion.renamed the {patient} to {displayWithAge} and {name} to {display} which is the patient name before age is concatenated